### ISSUE TITLE
new app-misc/tmsu-9999 proposal

### ISSUE THREAD

**Issue by ghost** [wichtounet/wichtounet-overlay#2]
I'd like to share with you some improvements I made to your `app-misc/tmsu-9999` ebuild:
* Avoided homepage redirect
* Moved from EAPI 5 to EAPI 6
* Moved from Mercurial to Git
* Added some missing binaries to list of installed files

I haven't learned Git yet, so I'm pasting my new ebuild here:

```
# Copyright 1999-2016 Gentoo Foundation
# Distributed under the terms of the GNU General Public License v2
# $Header: $

EAPI=6

DESCRIPTION="TMSU: Tagging file management system"
HOMEPAGE="http://tmsu.org/"

inherit eutils git-r3

LICENSE="MIT"
SLOT="0"
KEYWORDS="~amd64"
IUSE=""

RDEPEND="
    >=dev-lang/go-1.1.1
    =dev-go/fuse-9999
    =dev-go/sqlite3-9999"

DEPEND="
    dev-vcs/git
    ${RDEPEND}"

EGIT_REPO_URI="https://github.com/oniony/TMSU"

src_compile(){
    emake
}

src_install() {
    dobin bin/tmsu
    dobin misc/bin/tmsu-fs-merge
    dobin misc/bin/tmsu-fs-mv
    dobin misc/bin/tmsu-fs-rm
    dosbin misc/bin/mount.tmsu
    doman misc/man/tmsu.1

    insinto /usr/share/zsh/site-functions/
    newins misc/zsh/_tmsu _tmsu
}
```

**Comment by wichtounet** [comment]
Hi, 

It looks fine and much better than mine. Thank you :+1: 

If you don't know Git, you can simply go here: https://github.com/wichtounet/wichtounet-overlay/blob/master/app-misc/tmsu/tmsu-9999.ebuild and use the Edit button (the pen) and it should create a Pull Request for you directly. I'll then merge it and regenerate the Manifest. At least you'll mentioned as contributor. 

If you prefer, I can also integrate it myself.

**Comment by wichtounet** [comment]
I've merged the Pull Request #3 and updated the Manifest. 

Thank you :)
